vendor/engines/bulk_email/app/views/bulk_email/bulk_email/_fields.html.haml

```diff
@@ -19,7 +19,7 @@
   .row
     %h4.span8= t("bulk_email.dates.label")
 
-  .row
+  #dates_between.row
```

Comment:
I think we lost this `id` in 8c91b7d7a788b3a9def5339e5f3f48069aa48244. It might be better as a `.js--*` class now but this should restore the enable/disable date field behavior without changing the javascript for now (it's going to change in a near-future PR).